specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json

```diff
@@ -12103,6 +12148,58 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/listbackups": {
```

Comment:
The APIs look fine, but what is the need to expose these as POST actions? I assume they are not privileged and there are equivalent GET apis already. Is there really a need for the alternate method?

Reply:
GETs right now expose secrets, so we need the post for RBAC.